frontend/src/components/core/StreamlitDialog/styled-components.ts

```diff
@@ -21,3 +21,9 @@ export const StyledUploadUrl = styled.pre(({ theme }) => ({
   whiteSpace: "normal",
   wordWrap: "break-word",
 }))
+
+export const StyledShortcutLabel = styled.span(({ theme }) => ({
+  "&::first-letter": {
+    textDecoration: "underline",
+  },
+}))
```

Comment:
I don't love the duping here. Shortcut util class object or a shared styled component?

Reply:
Yea. I think I create duplication cause it was just easier to reason, and it's only in two locations. I really want a well thought-out design system to organize these better. I thought duping would show the "pains"